app/views/topics/index.html.slim

```diff
@@ -26,9 +26,14 @@ div ng-controller="TopicsCtrl as topicsCtrl" class="animated fadeIn lanes"
         data-jqyoui-draggable="{index: {{$index}}, animate: true, onStop: 'topicStoppedDraggingCallback'}"
         class="topic"
         ]
+        small class="hover-options" ng-click="$event.stopPropagation()"
```

Comment:
Because the remove vote elements are nested inside the vote button. This line hinders the mouse click from also adding a vote. This might also be solved inside the controller, however the function in the controller should not be specific to the html in the view. So I put it here.